addons/website/static/src/js/form_editor_registry.js

```diff
@@ -58,7 +58,9 @@ FormEditorRegistry.add('send_mail', {
         type: 'char',
         required: true,
         string: _t('Recipient Email'),
-        defaultValue: 'info@yourcompany.example.com',
+        // There's a default value for this field, but it's computed in the form
+        // options because it should only be used if there isn't already a value
+        // and there's no value coming from a data-for.
```

Comment:
I don't understand why we need this change: is not this form_editor_registry about describing how the form should be created when selecting the "Send mail" option / adding a form in the page? In both those cases, we don't care about forcing the default value as the user is supposed to edit it, no?